src/sentry/api/endpoints/accept_organization_invite.py

```diff
@@ -15,7 +15,7 @@
 from sentry.utils import auth
 
 
-@region_silo_endpoint
+@control_silo_endpoint
 class AcceptOrganizationInvite(Endpoint):
```

Comment:
if AcceptOrganizationInvite going to be a control silo endpoint, does it need the organization_slug in the URL? #42138

Reply:
Yes, probably. Note that I didn't actually mark this one stable. This requires us to make some decisions about organization membership too and that got hairy quickly, so I've deferred for now.

Reply:
I suspect this does, actually, need to remain region silo.  What we're missing is the OrganizationMemberMapping class that will allow control silo access to a replicated OrganizationMember object (which must remain in the region silo).